packages/extension-browser/package.json

```diff
@@ -132,7 +132,7 @@
     "lint:js": "eslint . --cache --ext js --ext md --ext ts --ext tsx --ignore-path ../../.eslintignore --report-unused-disable-directives",
     "lint:dependencies": "node ../../scripts/lint-dependencies.js",
     "lint:md": "node ../../scripts/lint-markdown.js",
-    "prebuild:assets": "cpx \"./src/**/*.{html,json,png}\" dist/bundle && cpx \"./src/*.svg\" dist/bundle",
+    "prebuild:assets": "cpx \"./src/**/*.{html,json,png,svg}\" dist/bundle && cpx \"./src/*.svg\" dist/bundle",
```

Comment:
This results in redundant SVG files in our bundle as we inline most of them in the CSS so we don't typically want to copy the files themselves (This is likely why you had to increase the bundle size so much). The one exception are `*.svg` files directly under `src` (as seen later in this line of code).

I'd suggest moving `logo.svg` directly under `src` and reverting this particular change.

Reply:
After doing this, the size is 652KB.